modular_causticcove/code/game/objects/items/weapons/ranged/ammo.dm

```diff
@@ -0,0 +1,6 @@
+// Caustic Cove overrides of code/game/objects/items/rogueweapons/ranged/ammo.dm
+// so we can override or balance it how we want to.
+
+/obj/item/ammo_casing/caseless/rogue/bullet
+	name = "musket ball"
```

Comment:
I wonder if I should call them "musket ball" or just "arquebus bullet" and I'm not picky.

Reply:
I'm just going to leave it as-is for the time being since the majority of people OOCly know what a "musket ball" is but without context might see an "arquebus ball" and be like "wtf is this" and if I call it "bullet ball" they might think it's supposed to be a sling bullet.

"Lead sphere" is even worse because it's not even lead at this time.